javascript.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>Web_Page</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="Styles.css">
+  <script type="text/javascript" src="main.js"></script>  
+</head>
+<body onload="createbody()">
+  <header class="hd"></header>

```

Comment:
Please change class names, use proper naming convention.
https://en.bem.info/methodology/naming-convention/
http://bdavidxyz.com/blog/how-to-name-css-classes/